librz/core/cmd/cmd_debug.c

```diff
@@ -1628,7 +1699,8 @@ static void asciiart_backtrace(RzCore *core, RzList *frames) {
 	}
 }
 
-static void get_backtrace_info(RzCore *core, RzDebugFrame *frame, ut64 addr, char **flagdesc, char **flagdesc2, char **pcstr, char **spstr) {
+static void
+get_backtrace_info(RzCore *core, RzDebugFrame *frame, ut64 addr, char **flagdesc, char **flagdesc2, char **pcstr, char **spstr) {
```

Comment:
do not split this way. at most, split the argument list but keep the function name on the first line.